contentcuration/contentcuration/frontend/shared/vuex/session/index.js

```diff
@@ -85,20 +100,28 @@ export default {
     },
   },
   actions: {
+    async saveSession(context, currentUser) {
+      // It shouldn't happen but let's make sure that there is
+      // no more current users than one in the table (do not remove
+      // the current user being saved if it already exists in the table
+      // though, because dexie-observable delete listener handler would
+      // be run and depending on timing, it might remove the entry after
+      // it's added and the current user would be lost)
+      await Session.indexNotEqual('id', currentUser.id).delete();
+
+      await Session.put(currentUser);
```

Comment:
Would then need to make sure that we add `constantId: 'CURRENT'` to the `currentUser` object before we do the put here.